handler.go

```diff
@@ -111,7 +111,32 @@ func (s *RPCServer) handleReader(ctx context.Context, r io.Reader, w io.Writer,
 	}
 
 	var req request
-	if err := json.NewDecoder(r).Decode(&req); err != nil {
+	// Limit request size. Ideally this limit should be specific for each field
+	// in the JSON request but as simple defensive measure we just limit the
+	// entire HTTP body.
+	MAX_REQUEST_SIZE := 10 << 10 // FIXME: Agree on a value and extract.
```

Comment:
Some Worker calls to things which compute snarks send ~20-30MB, so I'd probably set this to 100M

Reply:
@magik6k are there any cases where 100MB+ would happen on requests?

Reply:
I'm also fine with this being a configurable option (with a sane default like 100 MiB) for the consumer to adjust as necessary. Not sure what would be the most appropriate way to implement that.

Reply:
Added it as a server configuration, will make Lotus consume it as a node options during dependency update.